scylla/src/transport/load_balancing/default.rs

```diff
@@ -1823,18 +1840,47 @@ mod latency_awareness {
             }
         }
 
+        /// Sets the scale to use for the resulting latency aware policy.
+        ///
+        /// The `scale` provides control on how the weight given to older latencies decreases
+        /// over time. For a given host, if a new latency `l` is received at time `t`, and
+        /// the previously calculated average is `prev` calculated at time `t'`, then the
+        /// newly calculated average `avg` for that host is calculated thusly:
+        ///
+        /// ```text
+        /// d = (t - t') / scale
+        /// alpha = 1 - (ln(d+1) / d)
+        /// avg = alpha * l + (1 - alpha * prev)
```

Comment:
This looks wrong, shouldn't it be `avg = alpha * l + (1 - alpha) * prev` ? It looks like the closing paren is too far.

Reply:
This is taken directly from Java driver 3.x docstring. Why do you think it is erroneous?

Reply:
It looks like it was supposed to represent exponential smoothing from `l` to `prev` with an `alpha` weight. This neither corresponds to our code:

```rust
let average = Duration::from_secs_f64(
    (1. - prev_weight) * last_latency_secs + prev_weight * prev_avg_secs,
);
```

nor the original code from the cpp driver:

```c++
current_.average =
    static_cast<int64_t>((1.0 - weight) * latency_ns + weight * previous.average);
```

The code is correct, the docstring is just wrong.

Reply:
I'll fix the docstring then. Should we bother with fixing the docstring in Java driver?

Reply:
@wprzytula Yes, please send the PR to Java Driver. The easier path is to send it only to our fork, but if you're willing to take some more time (dealing with "Contributor License Agreement") you can also send to the upstream - according to https://github.com/datastax/python-driver/pull/1114#issuecomment-953564923 we have already signed all the neccessary documents to do this.

Reply:
Done.